solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: ac-GCDOnBlackboard
+source: AC
+title: GCD On Blackboard
+author: Andrew Cheng
+---
+
+**Time Complexity:**
```

Comment:
probably better to move all the time complexity stuff after line 34

Reply:
^